ruby_hack_assembler_test.rb

```diff
@@ -0,0 +1,51 @@
+require "minitest/autorun"
+
+class RubyHackAssemblerTest < Minitest::Test
+  def test_integration_test
+    skip("Assembly with symbols hasn't been implemented!")
+    assembly_code = `ruby ruby_hack_assembler.rb examples/Rect.asm`
+    expected = <<~EOF
+      0000000000000000
+      1111110000010000
+      0000000000010111
+      1110001100000110
+      0000000000010000
+      1110001100001000
+      0100000000000000
+      1110110000010000
+      0000000000010001
+      1110001100001000
+      0000000000010001
+      1111110000100000
+      1110111010001000
+      0000000000010001
+      1111110000010000
+      0000000000100000
+      1110000010010000
+      0000000000010001
+      1110001100001000
+      0000000000010000
+      1111110010011000
+      0000000000001010
+      1110001100000001
+      0000000000010111
+      1110101010000111
+    EOF
+
+    assert_equal(expected, assembly_code)
+  end
+
+  def test_integration_test_symboless
+    assembly_code = `ruby ruby_hack_assembler.rb examples/Add.asm`
```

Comment:
Is this the right filename?

Reply:
<img width="374" alt="Screen Shot 2021-06-23 at 11 26 34 AM" src="https://user-images.githubusercontent.com/25471753/123124680-e0738180-d415-11eb-8e2c-3054cec9a3eb.png">

Ah interesting, I updated it locally but it seems like git didn't pick up on it?

Reply:
![image](https://user-images.githubusercontent.com/25471753/123125020-2af4fe00-d416-11eb-93de-5167b4ce86ff.png)

Used `git mv`

Reply:
Now fixed!